aten/src/ATen/native/mps/operations/Normalization.mm

```diff
@@ -282,13 +281,14 @@ Check if running mean exists (maybe do this check before making graph)
           }
 
           // Compute output of batch norm
-          MPSGraphTensor* outputTensor = [mpsGraph normalizationWithTensor:inputTensor
-                                                                meanTensor:saveMeanTensor
-                                                            varianceTensor:varTensor
-                                                               gammaTensor:weightTensor
-                                                                betaTensor:biasTensor
-                                                                   epsilon:(float)epsilon
-                                                                      name:nil];
+          const auto inputTensorType = [inputTensor dataType];
+          MPSGraphTensor* outputTensor = [mpsGraph normalizationWithTensor: inputTensor
+                                                                meanTensor: castMPSTensor(mpsGraph, saveMeanTensor, inputTensorType)
```

Comment:
The normalization shouldn't need casts here. The computation graph for normalization should be performed in the type  which is requested by the user. Even if it's a pass through, it can add spurious casts which should have been fixed earlier in the Graph to add casts at proper places. My main concern is that it may lead to actual casts in future and will be hidden from the next person modifying the code. I am curious do you need the casts here  for this crash ?